### PR TITLE
chore: add bitcoind-onchain to testflight

### PR DESCRIPTION
**Pull request by openoms**

### CODE REVIEW

File: ci/testflight/.gitignore
Comment (krtk6160) [inline ci/testflight/.gitignore:1]: Not sure if this is needed here, we don't usually run tf locally in this directory

File: dev/bitcoin/main.tf [hunk 1]
```diff
@@ -39,6 +39,9 @@ resource "kubernetes_secret" "bitcoind_smoketest" {
     bitcoind_endpoint    = "bitcoind.${local.bitcoin_namespace}.svc.cluster.local"
     bitcoind_port        = 18443
     bitcoind_user        = "rpcuser"
+
+    bitcoind_onchain_rpcpassword = local.bitcoind_rpcpassword
```
Comment (krtk6160) [inline dev/bitcoin/main.tf:43]: Should this be `local.bitcoind_onchain_rpcpassword` or something?

**Review by krtk6160 — CHANGES_REQUESTED** [review]
Couple of comments

**Review by krtk6160 — APPROVED** [review]

File: ci/testflight/.gitignore
Comment (openoms) [reply ci/testflight/.gitignore:1]: used it to validate locally

File: dev/bitcoin/main.tf (hunk 1 above)
Comment (openoms) [reply dev/bitcoin/main.tf:43]: reusing the same password

**Pull request merged**